app/assets/javascripts/application.js

```diff
@@ -18,6 +18,7 @@
 //= require jquery.fontselect
 //= require dataTables/jquery.dataTables
 //= require dataTables/bootstrap/3/jquery.dataTables.bootstrap
+//= require stat_slider
```

Comment:
why does stats slider need to be available to the proprietor page? It should only need to be loaded if the theme is selected?

Reply:
or is stat_slider something entirely different?